src/sentry/models/groupsubscription.py

```diff
@@ -98,7 +126,10 @@ def get_participants(self, group):
                     user__in=users,
                     key='workflow:notifications',
                     project=group.project,
-                    value=UserOptionValue.all_conversations,
+                    value__in=[
+                        UserOptionValue.all_conversations,
+                        UserOptionValue.no_conversations,
```

Comment:
actually this probably isnt needed since they already wouldn't have shown up in the previous query